appinventor/blocklyeditor/src/generators/yail.js

```diff
@@ -52,7 +52,7 @@ Blockly.Yail.YAIL_CLOSE_BLOCK = ")\n";
 Blockly.Yail.YAIL_COMMENT_MAJOR = ";;; ";
 Blockly.Yail.YAIL_COMPONENT_REMOVE = "(remove-component ";
 Blockly.Yail.YAIL_COMPONENT_TYPE = "component";
-Blockly.Yail.YAIL_CONSTANT_ALL = 'com.google.appinventor.components.runtime.util.YailDictionary:ALL';
+Blockly.Yail.YAIL_CONSTANT_ALL = '(static-field com.google.appinventor.components.runtime.util.YailDictionary \'ALL)';
```

Comment:
Should use double quotes here to get rid of escaping the single quote

Reply:
Since we're trying to follow the [Google style guides](https://google.github.io/styleguide/jsguide.html#features-string-literals), string literals are supposed to be single quoted, not double quoted (I realize that most of the file isn't actually following that convention at the moment). Note that the style guide recommends using a template string in this case, but because we are targeting ES5 that's not an available feature. I'm going to recommend that we stick with the single quotes for now. As we make updates to the sources we can migrate code to match the style guide as needed.